config/ompi_check_libfca.m4

```diff
@@ -25,41 +25,37 @@ AC_DEFUN([OMPI_CHECK_FCA],[
     OPAL_CHECK_WITHDIR([fca], [$with_fca], [lib/libfca.so])
```

Comment:
The help string is inaccurate -- it says it looks for libraries in DIR, but it really looks in DIR/lib (and headers in DIR/include).  Can you fix that up as part of this PR, too?  (sorry!)